src/scikit_build_core/resources/_editable_redirect.py

```diff
@@ -42,6 +43,11 @@ def __init__(
         self.build_options = build_options
         self.install_options = install_options
         self.dir = dir
+        self.install_dir = (
+            install_dir
+            if os.path.isabs(install_dir)
```

Comment:
If this is an absolute path, then this is basically broken. Absolute paths are an experimental feature that allow you to target special directories (like bin directories) - which won't support this finder anyway. Maybe we could make this an error for now.

Reply:
I was considering if they pass `--config-settings=wheel.install-dir=/absolute/path` overwrite in combination with `-e`.  Technically that combination could be allowed right? Super niche situation :shrug:

Reply:
The "correct" fix would be to map "/data", "/headers", "/scripts", "/null", and "/metadata" to the proper places on the system if possible. It comes for free in the editable wheel install step, but we've have to compute them ourselves here.

But making it an error (since it even requires the experimental flag) is fine for now.

Reply:
> Technically that combination could be allowed right?

The wheel install still goes through an editable wheel. And it will currently ask you to add the experimental flag, and if you do, it will then require the set of absolute paths listed above.

Reply:
If we have an "install" command in CLI, then yes, but that won't be a config-setting.

Reply:
> And it will currently ask you to add the experimental flag, and if you do

Aren't the editable installs out-of-experimental now?

Reply:
Using absolute paths for install-dir is an experimental feature which allows you to target data, headers, scripts, and metadata directories.